src/quo2/components/list_items/preview_list/properties.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.list-items.preview-list.properties)
+
+(defn border-type
+  [type]
+  (case type
+    (:accounts :collectibles) :squared
+    :rounded))
```

Comment:
Just a small comment, this could be an `if`, I'd use `case` for more complex checks, I prefer to use simpler functions when possible, but it's just a personal preference